apple/Tests/TestCases/Prelude/DependencyInformationTests.swift

```diff
@@ -6,6 +6,6 @@ import XCTest
 
 final class DependencyInformationTests: Test<DependencyInformation> {
 	func test_description() {
-		XCTAssertNoDifference(SUT.sample.description, "develop")
+		XCTAssertNoDifference(SUT.sample.description, "main")
```

Comment:
intentional (but not needed :P ) :D I wanted to limit number of hits on search for `develop`...